layers/meta-balena-raspberrypi/conf/layer.conf

```diff
@@ -131,6 +131,7 @@ RPI_KERNEL_DEVICETREE:append:raspberrypi3-64 = " broadcom/bcm2710-rpi-cm3.dtb"
 RPI_KERNEL_DEVICETREE:append:raspberrypi4-64 = " broadcom/bcm2711-rpi-400.dtb"
 RPI_KERNEL_DEVICETREE:append:raspberrypi4-64 = " broadcom/bcm2711-rpi-cm4.dtb"
 RPI_KERNEL_DEVICETREE:append:revpi-connect-s = " broadcom/bcm2711-rpi-cm4s.dtb"
+RPI_KERNEL_DEVICETREE:append:revpi-connect-4 = " broadcom/bcm2711-rpi-cm4.dtb"
```

Comment:
Is this needed considering the machine requires revpi4.inc which in turn includes raspberrypi4-64.conf ?

Reply:
I think this might be not needed, I haven't tested it without this.

Reply:
The image seems to be working without this.